bin/release

```diff
@@ -40,13 +40,21 @@ echo "========================="
 echo ""
 
 echo "Releasing chart..."
+
+# This is needed to avoid using the CI's built-in
+# gcloud configurations.
+export BOTO_CONFIG=/dev/null
```

Comment:
The is a conflict in the built-in gcloud that's already in CI and the binary installed by our script. This variable serves to avoid making use of Circle's configs, which were messing it up.